eng/dotnet.ps1

```diff
@@ -0,0 +1,39 @@
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+  [Parameter(ValueFromRemainingArguments=$true)][String[]]$properties
+)
+
+. $PSScriptRoot\common\tools.ps1
+
+try {
+  # Import custom tools configuration, if present in the repo.
+  # Note: Import in global scope so that the script set top-level variables without qualification.
+  $configureToolsetScript = Join-Path $EngRoot "configure-toolset.ps1"
+  if (Test-Path $configureToolsetScript) {
+    . $configureToolsetScript
```

Comment:
I think this should be imported before tools.ps1 if I understand the intention. At least, dotnet.sh and dotnet.ps1 should have the same behavior.

Reply:
Didn't make much sense to me either.  I think sourcing order doesn't actually matter as log as it happens before InitializeBuildTools.  Either way, thie code will be deleted.